### DISCUSSION TITLE
LocalPeopleResults not showing any results

### DISCUSSION THREAD

**Discussion by TobiasK4T** [microsoft-search/pnp-modern-search#3492] [Q&A] [answered]
Hello everyone,

since last week I am having trouble with the LocalPeopleResults in my PnP Results Webpart. The option is not showing any results even when i use a new Results Webpart and just pick the People Layout. When i am using a different Source ID, results are showing. 

Today, I updated PnP Modern Search Webpart from 4.9.1 to 4.10.2, without any success. The same issue is happening in a customer tenant where Version 4.8.0 is installed and it was working fine beforhand.

Anyone experiencing the same issue and has a solution to this? Did Microsoft change something with the People Results?

Please let me know if futher information is needed.

![grafik](https://github.com/microsoft-search/pnp-modern-search/assets/99403575/5310be3d-20e5-4279-adb8-a4c55c9ff4a7)

**Comment by kasperbolarsen** [comment, marked as answer]
Sharepoint people search is down on Microsofts side, so we'll just have to wait

**Reply by TobiasK4T** [reply]
Thank you for the fast response. Any experience or suggestion how long this will take?

**Comment by Luca3082** [comment]
Hi all, same problem with 3 different tenants. I'll follow this thread too...

**Comment by ShareCloudOY** [comment]
Hi everyone,
We are also facing the same issue using Search API - it does not deliver any result when filtering:

- /_api/search/query?querytext=**'contentclass:STS_User'**

Also we have tried to filter results by sourceId - no results:
- /_api/search/query?querytext='test'&**sourceid='b09a7990-05ea-4af9-81ef-edfab16c4e31'**

Does anyone have a solution for this?

**Reply by kasperbolarsen** [reply]
It is a outage on SharePoint people search , at least in Europe but perhaps worldwide. 
MS are working on a fix

**Comment by nimblemoe** [comment]
The Microsoft Graph People Search Webpart still works apparently:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/159041848/02fed5f0-8e0d-4adb-80ea-8009f061b311)

**Reply by kasperbolarsen** [reply]
Microsoft search,  aka Graph search,  is an independent solution and would not be impacted by a SharePoint search outage

**Comment by AndyTuke** [comment]
Looks like the service is back up
